tests/baselines/reference/ambientConstLiterals.js

```diff
@@ -74,5 +74,5 @@ declare const c9: {
 declare const c10: number[];
 declare const c11: string;
 declare const c12: number;
-declare const c13: string;
-declare const c14: number;
+declare const c13: "abc" | "def";
+declare const c14: 123 | 456;

```

Comment:
This seems to reintroduce the widening problem we've previously discussed.

Reply:
I did notice this, but I think this is actually a bug. Hovering over those variables in the [playground](https://www.typescriptlang.org/play/?#code/MYewdgzgLgBMCMBmGBeGBZAhlAFgOgCdMwATEAWwAoBKGAPhgAY8BWGAfhgCJMAjYLjABc3EgFMAZlwDcAWABQoSLAQAWVBmz4ipCjXpNWHGPABMyEapYA2aUA) shows the union on hover while the declarations show `string` and `number`.

Reply:
It's a case where the declaration file is going to be inaccurate whichever it picks. In reality, only something like `declare const c13 = var ? "abc" : "def"` would actually get the correct widening behavior, but we don't support that in declaration files, so we compromise and fall on the side of emitting the widened type. Is it a bug? Yes. Is this the fix? No. Is the fix in the scope of this PR? Definitely not.

Reply:
Removed the change.